messenger/make.py

```diff
@@ -1,112 +1,133 @@
-#!/usr/bin/python
 from __future__ import print_function
+
 import os
-import platform
 import sys
 import shlex
 import shutil
 import subprocess
+import platform
+import glob
+
+try:
+    from ConfigParser import ConfigParser
+except ImportError:
+    from configparser import ConfigParser
+
+from . import settings
 
+__all__ = [
+    'get_messenger_dir',
+    'get_matlab_bin',
+    'get_config',
+    'do_build',
+    'build_octave',
+    'build_matlab'
+]
 
 def get_messenger_dir():
-    # Check the system platform first
-    splatform = sys.platform
-
-    if splatform.startswith('linux'):
-        messenger_dir = 'mexa64'
-    elif splatform.startswith('darwin'):
-        messenger_dir = 'mexmaci64'
-    elif splatform.startswith('win32'):
-        if splatform == "win32":
-            # We have a win64 messenger, so we need to figure out if this is 32
-            # or 64 bit Windows:
-            if not platform.machine().endswith('64'):
-                raise ValueError("pymatbridge does not work on win32")
-
-        # We further need to differniate 32 from 64 bit:
-        maxint = sys.maxsize
-        if maxint == 9223372036854775807:
-            messenger_dir = 'mexw64'
-        elif maxint == 2147483647:
-            messenger_dir = 'mexw32'
-    return messenger_dir
-
-
-def get_config():
-    messenger_dir = get_messenger_dir()
-    with open(os.path.join(messenger_dir, 'local.cfg')) as fid:
-        lines = fid.readlines()
+    host, is_64bit = platform.system(), sys.maxsize > 2**32
```

Comment:
This will reintroduce a problem I had running `pymatbridge` with 32 bit python on 64 bit Windows.
If you are running 32 bit python on 64 bit windows `sys.maxsize > 2**32` gives `False` while `platform.machine().endswith('64')` returns `True`.